src/dlgConnectionProfiles.cpp

```diff
@@ -1609,10 +1609,10 @@ void dlgConnectionProfiles::loadProfile(bool alsoConnect)
     QDir dir(folder);
     dir.setSorting(QDir::Time);
     QStringList entries = dir.entryList(QDir::Files, QDir::Time);
-    bool preInstallPackages = false;
+    bool loadingForFirstTime = false;
```

Comment:
🤔 It is a little unclear, but this actually means "loadingThisProfileForFirstTime" - given that that is a little long, can we add a comment to that effect here instead?

Reply:
Sure, done.